modules/@apostrophecms/ui/ui/apos/components/AposToggle.vue

```diff
@@ -18,6 +18,10 @@ export default {
     modelValue: {
       type: Boolean,
       required: true
+    },
+    noFocus: {
```

Comment:
these props are usually named `focus` or `disableFocus`... does it make sense to rename this?